File: day-01/python/part1.py
import sys


def main():
    a = 0
    amax = 0
    for l in sys.stdin:
        l = l.rstrip()
        if l == '':
            amax = max(amax, a)
            a = 0
        else:
            a += int(l)
    print(amax)


if __name__ == "__main__":
    main()

File: day-01/julia/part1.jl
#!/usr/bin/julia


function solve()
    calories = 0
    max_calories = 0
    for line in eachline(stdin)
        if line == ""
            max_calories = max(max_calories, calories)
            calories = 0
        else
            calories = calories + parse(Int, line)
        end
    end
    println(max_calories)
end

solve()
